flink-connector-mongodb/src/main/java/org/apache/flink/connector/mongodb/source/split/MongoScanSourceSplitState.java

```diff
@@ -23,18 +23,19 @@
 
 /** MongoDB source split state for {@link MongoScanSourceSplit}. */
 @PublicEvolving
```

Comment:
this can now be `@Internal`?

Reply:
Changed MongoSourceSplitState and MongoScanSourceSplitState from PublicEvolving to Internal.